contrib/msggen/msggen/schema.json

```diff
@@ -23999,7 +23999,25 @@
                 "max_total_htlc_in_msat": {
                   "type": "msat",
                   "description": [
-                    "Max amount accept in a single payment."
+                    "Max amount accept in a single payment. This field is deprecated, use instead our_max_htlc_value_in_flight_msat"
+                  ],
+                  "deprecated": [
+                    "v24.11",
+                    "v25.05"
+                  ]
+                },
+                "their_max_htlc_value_in_flight_msat": {
+                  "type": "msat",
+                  "added": "v24.11",
+                  "description": [
+                    "Cap on total value of outstanding HTLCs offered by the remote node. This limits the total amount in flight we can send through this channel."
```

Comment:
suggestion to make this more readable: `cap on total value of outstanding HTLCs offered to the remote node (from our balance).`

Reply:
though now that i've written these out, i feel like the `their` part of the max htlc value in flight name is confusing, since it's our balance that's in flight? something about this seems wrong. Either the modifier "this limits the total amount in flight we can send" is wrong or `their max in flight` is incorrect?

we can't add more HTLCs to be sent to the peer, because they don't want to let X% of their balance be committed as HTLCs?

Reply:
Yes, this is weird. It is like `our_max_value...`  is telling the peer "please don't try to send me more than this
or I will not accept it". The prefix `their_` and `our_` refers to who set the configuration value.

Reply:
The description is a sentence taken from BOLT02.